docs/source/documentation/platform/infrastructure/containers.html.md.erb

```diff
@@ -0,0 +1,78 @@
+---
+owner_slack: "#data-platform-notifications"
+title: Containers
+last_reviewed_on: 2023-07-25
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+Data Platform's Core Infrastructure team offer a managed pipeline for building, scanning and pushing containers to a registry. This is done via GitHub Actions and is available to all teams.
+
+Images are scanned for vulnerabilities using [Trivy](https://github.com/aquasecurity/trivy), currently with a default severity of `CRITICAL`
+
+Dockerfiles are linted by the Super Linter with [Hadolint](https://github.com/hadolint/hadolint)
+
+## Creating a new container
+
+To create a new container, you will need to:
+
+1. Create a directory in `containers`, e.g. `containers/example-application`
+
+1. Create the configuration file `containers/${CONTAINER_NAME}/config.yml` with the following contents
```

Comment:
Minor question - all these show as 1. -  assume this is a templating thing and will be fine when rendered?

Reply:
Yes, it will render out as a proper list!